### PR TITLE
Fix a bug and run the optimisation with nonzero ascending nodes

### PR DESCRIPTION
**Pull request by pleroy**
```
E0608 22:11:41.584633  2896 trappist_dynamics_test.cpp:635] Best : χ² = 478.579787 max Δt = 1265.202394 s (Trappist-1g[7294.786000])
 avg Δt = 64.134682 s
E0608 22:11:41.585633  2896 trappist_dynamics_test.cpp:1201] The Blind Idiot God
E0608 22:11:41.585633  2896 trappist_dynamics_test.cpp:1203] Trappist-1b
E0608 22:11:41.585633  2896 trappist_dynamics_test.cpp:1204] {e = +4.83076901277041879e-03, T = +1.30531598576478107e+05 s, i = +1.5
6311687808612154e+00 rad, Ω = +3.23214033518325864e-04 rad, ω = +2.27612602906071526e+00 rad, M = +8.94140783919841908e+00 rad}
E0608 22:11:41.586633  2896 trappist_dynamics_test.cpp:1203] Trappist-1c
E0608 22:11:41.586633  2896 trappist_dynamics_test.cpp:1204] {e = +1.02229527025948924e-02, T = +2.09257288444493082e+05 s, i = +1.5
6556033903891367e+00 rad, Ω = -2.60019151962115206e-04 rad, ω = -1.44879092705289536e+00 rad, M = +1.07379537897302910e+01 rad}
E0608 22:11:41.586633  2896 trappist_dynamics_test.cpp:1203] Trappist-1d
E0608 22:11:41.586633  2896 trappist_dynamics_test.cpp:1204] {e = +1.02900864761148025e-02, T = +3.49830223331187444e+05 s, i = +1.5
6887646461770291e+00 rad, Ω = -6.68708940267611447e-04 rad, ω = -2.69117006105044165e+00 rad, M = +7.49021276134394043e+00 rad}
E0608 22:11:41.587633  2896 trappist_dynamics_test.cpp:1203] Trappist-1e
E0608 22:11:41.587633  2896 trappist_dynamics_test.cpp:1204] {e = +8.43434192829547145e-03, T = +5.27009671423300984e+05 s, i = +1.5
6618865756963155e+00 rad, Ω = -9.98733248527219116e-04 rad, ω = -1.26467718365020865e+00 rad, M = +7.43665522173231519e+00 rad}
E0608 22:11:41.587633  2896 trappist_dynamics_test.cpp:1203] Trappist-1f
E0608 22:11:41.587633  2896 trappist_dynamics_test.cpp:1204] {e = +1.35674982095662922e-02, T = +7.95669716029515956e+05 s, i = +1.5
6589195159679240e+00 rad, Ω = -5.63497002298889231e-05 rad, ω = -2.33226321965278549e+00 rad, M = +1.06799175145282366e+01 rad}
E0608 22:11:41.588634  2896 trappist_dynamics_test.cpp:1203] Trappist-1g
E0608 22:11:41.588634  2896 trappist_dynamics_test.cpp:1204] {e = +3.20594397404127141e-03, T = +1.06722255987686105e+06 s, i = +1.5
6592685818183242e+00 rad, Ω = -8.26544300513214702e-05 rad, ω = -1.26823764692949381e+00 rad, M = +9.99156853379950149e+00 rad}
E0608 22:11:41.588634  2896 trappist_dynamics_test.cpp:1203] Trappist-1h
E0608 22:11:41.588634  2896 trappist_dynamics_test.cpp:1204] {e = +1.06435213309643423e-02, T = +1.62178079943851521e+06 s, i = +1.5
6723585512082830e+00 rad, Ω = +6.88162380110340205e-04 rad, ω = -1.46348376285032566e+00 rad, M = +7.40992609965667803e+00 rad}
```

### CODE REVIEW

File: astronomy/trappist_preoptimisation_initial_state_jd_2457000_000000000.proto - Copy.txt [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Most of the data are from  "The nature of the TRAPPIST-1 exoplanets", Grimm et
```
Comment (eggrobin) [inline astronomy/trappist_preoptimisation_initial_state_jd_2457000_000000000.proto - Copy.txt]: I'm not convinced by the extension `.proto - Copy.txt`.

File: astronomy/trappist_dynamics_test.cpp [hunk 2]
```diff
@@ -1049,7 +1049,8 @@ TEST_F(TrappistDynamicsTest, DISABLED_Optimisation) {
   SolarSystem<Sky> const system(
       SOLUTION_DIR / "astronomy" / "trappist_gravity_model.proto.txt",
       SOLUTION_DIR / "astronomy" /
-          "trappist_initial_state_jd_2457000_000000000.proto.txt");
+          "trappist_preoptimisation_initial_state_jd_2457000_000000000.proto."
+          "txt");
```
Comment (eggrobin) [inline astronomy/trappist_dynamics_test.cpp]: Fold the line *before* one of the `.`, `.txt` or `.proto.txt` are recognizable as extensions, whereas a full-stop terminated filename looks weird.

File: astronomy/trappist_preoptimisation_initial_state_jd_2457000_000000000.proto - Copy.txt (hunk 1 above)
Comment (pleroy) [reply astronomy/trappist_preoptimisation_initial_state_jd_2457000_000000000.proto - Copy.txt]: Done.

File: astronomy/trappist_dynamics_test.cpp (hunk 2 above)
Comment (pleroy) [reply astronomy/trappist_dynamics_test.cpp]: Done.

**Pull request merged**